### DISCUSSION TITLE
About concept of "public" channels.

### DISCUSSION THREAD

**Discussion by sguinetti** [iptv-org/community#110] [Q&A] [answered]
Greetins. The term PUBLIC exists in the CONTRIBUTING.md for adding new channels: "Understand that our community of volunteers will try to help you, but if a public link cannot be found, there is little we can do.". However, in order to avoid any misunderstanding, what would be the concept that fits with the word PUBLIC?

I understand that examples of links are public access when they come from open signal TV, free video platforms. official website, satellite broadcasting without subscription blocking and so on. Also for this stream to be a public url it must have open access without payment barriers or exclusivity in at least one country. Do you agree with this concept?

**Comment by sguinetti** [comment, marked as answer]
This can be corrected by creating the FAQ.

**Reply by dct-infra** [reply]
Hey @sguinetti 
_must have open access without payment barriers, **authentication mechanisms** or exclusivity in at least one country._
Since access can be free, an url can be open (acessible) and the resource restricted.
